packages/list-reusable-blocks/package.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "@wordpress/list-reusable-blocks",
+	"version": "1.0.0",
+	"description": "Adding Export/Import support to the reusable blocks listing.",
```

Comment:
I made it private :)

Reply:
Now that I think about it. Making it private means it can't be included in Core as an external package and begs the question of whether it should be moved to Core or kept in the repo post-merge.

Reply:
yes, I know it was private even before the comment :)